### ISSUE TITLE
[Translations Phase 2][Accessibility] 'Translate Page to...' panel fails to update font scaling when Dynamic Text size changes

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#33343]
### **Steps to Reproduce**

1. Launch Firefox.
2. Translate any webpage.
3. Tap to open ‘Translation’ panel from URL bar/three-dot menu.
4. In Accessibility, turn on Large Dynamic text and change the size.
5. Observe ‘Translate Page to…’.

### **Expected Behavior**

The text within the Translation panel should **dynamically scale** in real-time to match the new system font size.

### **Actual Behavior**

The panel fails to re-render; the text remains "stuck" at the previous size (appearing either too small or too large), unless the panel is dismissed and re-opened.

### **Device & Build Information**

- Device: iPhone 14 Pro Max (iOS 26.2)
- Build version: v9000 (70646)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-15561)

**Comment by data-sync-user** [comment]
> **Razvan Litianu** commented from Jira:

[~accountid:624be0db5f63fd0069b5a77a] We’re removing the bold title so this should not replicate anymore after the PR is merged

**Comment by data-sync-user** [comment]
> **Ada Cristea** commented from Jira:

The issue is still reproducible on Firefox v150.2 (71196).
